ui/src/App.tsx

```diff
@@ -25,6 +25,7 @@ function App() {
             <Route path="*" element={<EnterScreen />}>
               <Route path="incarnations" element={<IncarnationsList />} />
               <Route path="incarnations/create" element={<IncarnationsCreateForm />} />
+              <Route path="*" element={<Navigate to="/incarnations" />} />
```

Comment:
hmm, is this a wildcard or just a default matcher? e.g. would `/foobar` just redirect to `/incarnations`?

Reply:
yes, any unregistered path will redirect to `/incarnations`